google/cloud/firestore_v1beta1/order.py

```diff
@@ -49,7 +49,7 @@ def from_value(value):
         }
 
         if v not in lut:
-            raise ValueError("Could not detect value type for " + v)
+            raise ValueError("Could not detect value type for " + str(v))
```

Comment:
Same comment about f-strings.

Reply:
we could likely use f-strs now that we are dropping 2.7 support. I can add that here. Also going to open an issue to do general 2.7 cleanup.

Reply:
Sounds good! With your most recent commit it looks like `firestore_v1/order.py` is fixed, but `firestore_v1beta1` is the same. As this is non-breaking it is probably fine to leave for the open issue, but I figured I would note in case this is something that would be good to fix for this version.

I can also take a closer pass at 2.7 cleanup when working on async.